src/components/SecondaryPanes/Breakpoints.js

```diff
@@ -80,11 +79,7 @@ function renderSourceLocation(source, line, column) {
     return null;
   }
 
-  return (
-    <div className="location">
-      {`${endTruncateStr(filename, 30)}: ${bpLocation}`}
-    </div>
-  );
+  return <div className="location">{`$filename}: ${bpLocation}`}</div>;
```

Comment:
Not really a fan of putting everything in one line as it makes it really difficult to read, besides - we might be missing an opening `{` here.